poi/src/main/java/org/apache/poi/hssf/util/HSSFColor.java

```diff
@@ -324,11 +348,22 @@ public short getIndex2() {
      */
 
     public String getHexString() {
-        return (Integer.toHexString(color.getRed()*0x101) + ":" +
-               Integer.toHexString(color.getGreen()*0x101) + ":" +
-               Integer.toHexString(color.getBlue()*0x101)).toUpperCase(Locale.ROOT);
+        return (Integer.toHexString(getRed()*0x101) + ":" +
+               Integer.toHexString(getGreen()*0x101) + ":" +
+               Integer.toHexString(getBlue()*0x101)).toUpperCase(Locale.ROOT);
     }
 
+    final short getBlue() {
```

Comment:
would there a be a benefit to making these methods public?

So far, you don't appear to use these methods outside this class, so they could even be made private.

Reply:
- I didn't want to change the API - unless necessary
- there are no usages of these methods outside of this class thus I am making them private in 449dba6
- the `color` field wasn't publicly accessible either (luckily)